plugins/repository-s3/src/main/java/org/opensearch/repositories/s3/S3BasicSessionCredentials.java

```diff
@@ -31,22 +31,21 @@
 
 package org.opensearch.repositories.s3;
 
-import com.amazonaws.auth.AWSSessionCredentials;
+import software.amazon.awssdk.auth.credentials.AwsSessionCredentials;
 
 import java.util.Objects;
 
-final class S3BasicSessionCredentials extends S3BasicCredentials implements AWSSessionCredentials {
+final class S3BasicSessionCredentials extends S3BasicCredentials {
 
-    private final String sessionToken;
+    private final AwsSessionCredentials sessionCredentials;
 
     S3BasicSessionCredentials(String accessKey, String secretKey, String sessionToken) {
         super(accessKey, secretKey);
-        this.sessionToken = sessionToken;
+        sessionCredentials = AwsSessionCredentials.create(accessKey, secretKey, sessionToken);
```

Comment:
Curious why did we need this change. `AwsSessionCredentials` is just a wrapper https://github.com/aws/aws-sdk-java-v2/blob/master/core/auth/src/main/java/software/amazon/awssdk/auth/credentials/AwsSessionCredentials.java

Reply:
Yeah we don't seen `AwsSessionCredentials` here. I've pushed a change for this.